demo/addons/ropesim/Rope.gd

```diff
@@ -41,6 +43,12 @@ signal on_point_count_changed()
 ## Gravity direction. Will not be normalized.
 @export var gravity_direction: Vector2 = Vector2.DOWN
 
+## Enables and disables wind simulation.
+@export var wind_enabled: bool = false
+
+## Strength of wind force to keep it in a particular direction.
```

Comment:
Update doc string to fit the property, e.g. "Optional parameters for wind simulation."